submissions/NwagboJohn/HtmlTask/signUp.html

```diff
@@ -0,0 +1,34 @@
+
+   <!DOCTYPE html>

```

Comment:
Please check out the html style guide [here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/blob/master/STYLEGUIDE.md#html) on how to format your html files.